src/components/CampaignInitialisation/CampaignInitialisationScreen.tsx

```diff
@@ -105,21 +104,21 @@ export const CampaignInitialisationScreen: FunctionComponent<NavigationProps> =
     if (updateCampaignConfigError) {
       if (updateCampaignConfigError instanceof CampaignConfigError) {
         Sentry.captureException(updateCampaignConfigError);
-        showAlert({
-          ...systemAlertProps,
-          description: ERROR_MESSAGE.CAMPAIGN_CONFIG_ERROR
+        showErrorAlert({
+          translationKey: getTranslationKeyFromMessage(
```

Comment:
change showXXXAlerts to take in error object. Translation key abstract into context

Reply:
This is done!